python/pynessie/cli.py

```diff
@@ -46,10 +46,22 @@
 class ContextObject(object):
     """Click context object."""
 
-    nessie: NessieClient
+    config: confuse.Configuration
     verbose: bool
     json: bool
 
+    _nessie: Optional[NessieClient] = None
+
+    def nessie(self: "ContextObject") -> NessieClient:
```

Comment:
Now I am not sure why we have to lazy initialize the nessie object. We can't change the config after the ctx is created.

This change also means we should no longer use `attr.s` on the `ContextObject`. `attrs` library is meant for describing simple data objects and not really meant for stateful objects. At least imho. 

So we either have to:
1) remove the mutable state `_nessie`
2) remove the `attr.s` decorator and make this a normal class (w/ `__init__` and properties)